### DISCUSSION TITLE
Someone & Knux's Seperate Super & Hyper Sprites

### DISCUSSION THREAD

**Discussion by SomeoneNKnuckles** [SomeoneNKnuckles/Someone-s-Mods#8] [General]
![HOW](https://github.com/user-attachments/assets/9b4636ea-d586-48ff-848a-94c70e9c8f1b)

Tell Me What Other Things Could I Add To It
Or If I Should Publish It On Gamebanana

**Comment by Mazher94** [comment]
Just gonna say one thing.
Please no spaghetti code.
Aside from that, Cool!

**Reply by Mazher94** [reply]
Update: so I just checked it and saw that this version is missing some features like the super and hyper peelout sprites and the waiting animation for Super and hyper.
Those are really the only things I noticed that were missing, there could be more that I haven't noticed yet.
So far, I can see this being a great alternative to the daKing's seperate super and hyper sprites. All that it needs is just more support and updates.

Also, I'm excited for the hyper monitor icon! Can't wait to see what people do with it.

And that's all I wanted to say! Good luck on your version of seperate super and hyper sprites.

**Comment by Dazzle-Isle-CD** [comment]
This mod is peak
